qutebrowser/browser/commands.py

```diff
@@ -851,7 +851,7 @@ def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
                        count='count')
-    def tab_move(self, direction: {'type': ('+', '-')}=None, count=None):
+    def tab_move(self, direction: {'type': ('+', '-', str)}=None, count=None):
```

Comment:
The `type` annotation here doesn't really do anything helpful anymore, as it takes any string now anyways - so let's just remove it.